### PR TITLE
Update meeting-minute link in README.md

### LINKED ISSUE #307: Invalid `MeetingMinutes` - reference in README.md
**Problem description**
The following Meeting Minutes reference in the README.md is invalid:
https://wiki.camaraproject.org/display/CAM/Quality+on+Demand+Meeting+Minutes

**Expected action**
Change it to: 
https://wiki.camaraproject.org/display/CAM/QualityOnDemand+2024+Minutes

**Additional context**

### PR DESCRIPTION
**Pull request by maxl2287**
#### What type of PR is this?

* subproject management

#### What this PR does / why we need it:
Fixes the meeting-minute link to confluence.

#### Which issue(s) this PR fixes:

Fixes #307

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,15 +16,15 @@ Repository to describe, develop, document and test the QualityOnDemand API famil
...
+  * get notification if the network cannot fulfill  
+* Describe, develop, document and test the APIs (with 1–2 Service Providers)  
 * Started: October 2021
 * Location: virtually  
 
 ## Meetings
 
 * Meetings are held virtually: [Meeting registration / Join](https://zoom-lfx.platform.linuxfoundation.org/meeting/94112812156?password=f238d6af-c959-48d7-a862-abdb3c648e40)
-* Schedule: bi-weekly, Friday, 2 PM CET/CEST (13:00 UTC, 12:00 UTC during European DST). For date/time of next meeting see previous [meeting minutes](https://wiki.camaraproject.org/display/CAM/Quality+on+Demand+Meeting+Minutes).
+* Schedule: bi-weekly, Friday, 2 PM CET/CEST (13:00 UTC, 12:00 UTC during European DST). For date/time of the next meeting, see previous [meeting minutes](https://wiki.camaraproject.org/display/CAM/QualityOnDemand+2024+Minutes).
```
Comment (hdamker) [inline README.md:27]:
```suggestion
* Schedule: bi-weekly, Friday, 2 PM CET/CEST (13:00 UTC, 12:00 UTC during European DST). For date/time of the next meeting, see previous [meeting minutes](https://wiki.camaraproject.org/x/0AOeAQ).
```

**Review by hdamker — CHANGES_REQUESTED** [review]
Thanks for the PR, good catch ... see just my proposal regarding the link (created with "Share", making it independent of name changes).

**Comment by hdamker** [comment]
@maxl2287 Found another link which needed correction, done in https://github.com/camaraproject/QualityOnDemand/pull/308/commits/2a17108bdb98f90dd8c99bb2309890b98d7cecf8

**Review by RandyLevensalor — APPROVED** [review]
Thanks for updating the links and the additional cleanup.

**Pull request merged**